main.js

```diff
@@ -0,0 +1,148 @@
+var currentActivity;
+var pastActivities = [];
+
+var activityButtons = document.querySelectorAll(".button");
+var timerContainer = document.querySelector(".minutes-seconds-container");
+var selectActivityButton = document.querySelector(".button-container");
+var accomplishInputField = document.querySelector(".accomplish-input");
+var minInputField = document.querySelector("#minutesInput");
+var secInputField = document.querySelector("#secondsInput");
+var startActivityButton = document.querySelector(".start-activity-button");
+var studyIcon = document.querySelector("#study-icon");
+var meditateIcon = document.querySelector("#meditate-icon");
+var exerciseIcon = document.querySelector("#exercise-icon");
+var studyButton = document.querySelector("#studyButton");
+var meditateButton = document.querySelector("#meditateButton");
+var exerciseButton = document.querySelector("#exerciseButton");
+var defaultForm = document.querySelector('#defaultForm');
+var activityHeader = document.querySelector('#activityType');
+var startActivityForm = document.querySelector('#startActivityForm')
+var userDescriptionInput = document.querySelector('#userDescriptionInput')
+var startButton = document.querySelector('.start-button');
+var categoryError = document.querySelector('#categoryError');
+var descriptionError = document.querySelector('#descriptionError');
+var minutesError = document.querySelector('#minutesError');
+var secondsError = document.querySelector('#secondsError');
+var errorMessages = document.querySelectorAll('.error-message');
+var timer = document.querySelector('#timerInsert');
+
+startActivityButton.addEventListener("click", startActivity);
+
+startButton.addEventListener("click", function() {
+  currentActivity.startTimer();
+});
+
+selectActivityButton.addEventListener("click", function(event) {
+  var button = event.target.id;
```

Comment:
if we remove .id from this variable we can use some of our hide/show functions on line 40 43 and 46

Reply:
This has been updated as well using the refactored dynamic show/hide functions.